src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-flags-modal/osd-flags-modal.component.html

```diff
@@ -16,7 +16,8 @@
                  (change)="flag.value = !flag.value"
                  [name]="flag.code"
                  [id]="flag.code"
-                 [disabled]="flag.disabled">
+                 [cdPermission]="'osd'"
```

Comment:
I'm not sure ```cdPermission``` keyword is explicit/meaningful enough...
What about ```cdAllowEdit``` or sth like this?

Reply:
Yes. cdPermission is not making much sense. I just madeup while I was just testing and went along with it. I will change it into `cdAllowEdit` or something similar, thanks

Reply:
> Yes. cdPermission is not making much sense. I just madeup while I was just testing and went along with it. I will change it into `cdAllowEdit` or something similar, thanks

You can check with other contributors to find a consensual name.

Reply:
I've changed the name to `cdDisableEdit`.